extensions/omni_schema/src/make_plan.sql

```diff
@@ -0,0 +1,129 @@
+create or replace function make_plan(path text) 
```

Comment:
The code here assumes `local_fs`, however the intention is that we pass VFS instances to make sure it works off any source. You can see it in other places like `assemble_schema`.

`omni_vfs` interface is still a bit clunky so it is understandably a bit more difficult to work with it when it comes to remote connections. This is why `assemble_schema` copies from a local vfs into remote table_fs.

I wonder how difficult will it be for us to make it take fs at this point before rolling out the next release.

Longer-term we just need a good solution to cross-connection FS. Perhaps it might be not too difficult to have a `remote_fs` that goes over fdw?

Reply:
https://github.com/omnigres/omnigres/pull/755